src/Product.php

```diff
@@ -1071,10 +1073,6 @@ protected function processTags(array $data)
         }
 
         foreach ($data as $tag) {
-            if ($tag['tagType'] !== 'variation') {
```

Comment:
This is now supported?

Reply:
That is also fixed in 263 xD

Reply:
Still the same change after merge, how was it fixed in 263?

Reply:
With the new route, the assigned tags that are assigned to variants are automatically of type variation. Therefore they've dropped this field entirely. See this response for example:

![image](https://user-images.githubusercontent.com/20555112/92238516-b9b18200-eeb9-11ea-8d77-a718ab218c28.png)